content/en/blog/2022/knative.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Distributed tracing in Knative
+date: 2022-04-12
+author: Pavol Loffay
+---
+
+In this article, you will learn how distributed tracing works in Knative and we will explore how the OpenTelemetry project can make tracing support in this environment easier. We will explore Knative under the hood to understand what distributed tracing capabilities it provides out-of-the-box and which parts of the system need additional instrumentation.
+
+In this article we will not cover Knative fundamentals, please refer to the [Knative documentation](https://knative.dev/docs/) to get familiar with the project.
+
+## About Knative
+
+Knative is a serverless platform built on top of Kubernetes as a set of `CustomResourceDefinitions` (CRDs). The project is split into two logical parts: serving and eventing. Serving facilitates the creation, deployment and scaling of workload/services, and eventing facilitates event-driven communication between workloads to enable loosely coupled architectures.
```

Comment:
add a link to the k8s documentation on CRDs?

Reply:
There is no a single link listing all CRDs. The blog post does not focus on explaining Knative concepts and the first paragraph has a link to knative docs where people can get familiar with the project.

Reply:
Sure, it does not have to be explained here, I was just thinking about something like:
```markdown
Knative is a serverless platform built on top of Kubernetes as a set of [CustomResourceDefinitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) (CRDs).
```